security/middleware.py

```diff
@@ -13,6 +13,8 @@
 from django.utils import timezone
 import django.views.static
 
+from user_agents import parse
```

Comment:
`user-agents` uses https://github.com/ua-parser/uap-python under the hood, maybe we should just use that directly?


```
>>> from ua_parser import user_agent_parser
>>> user_agent_parser.ParseUserAgent(ua_string)
{
    'family': 'Chrome',
    'major': '41',
    'minor': '0',
    'patch': '2272'
}
```